### ISSUE TITLE
How to configure credentials

### ISSUE THREAD

**Issue by mxmlnkn** [fsspec/dropboxdrivefs#20]
I read the ReadMe and saw this issue: https://github.com/fsspec/dropboxdrivefs/issues/2
But how do I specify that token? I have tried: `fsspec.open("dropbox:///path?token=123)` but that did not work.

Furthermore, I am confused about which token to use. After creating the app, I also have the App key, App secrete, and the very long OAuth 2 token with the "do not share" warning. I assume that this one is to be used? What then are the app key and secret for? This sounds like something that could also be used for access and could be shipped with an app.

**Comment by MarineChap** [comment]
Indeed, the token you need to use is the OAuth2 token obtained on your docker app. 

There is probably a way with app key/secret but I did not explore it.

**Comment by mxmlnkn** [comment]
Thanks, that seems to work. It seems there is no way to specify the token with the URL scheme. It would be nice if there was an environment variable `DROPBOX_TOKEN` or maybe a file `~/.config/dropbox.token` that would be used.

Another issue I had is that I was failing to list the root directory because `ls("/")` does not work. `ls("")` does work though. I think this implementation detail o the dropbox SDK should be hidden by dropboxdrivefs.

```
dropbox.exceptions.BadInputError: BadInputError('12345',
'Error in call to API function "files/list_folder": 
request body: path: Specify the root folder as an empty string rather than as "/".')
```

```python3
import os
from dropboxdrivefs import DropboxDriveFileSystem as DFS

fs = DFS(token=os.environ["DROPBOX_TOKEN"])

print(fs.stat("/single-file.tar"))
print(fs.ls(""))
print(fs.ls("/"))
```

**Comment by mxmlnkn** [comment]
Btw, having to use that token, which seems to only have read access to the corresponding app folder, does that mean that this project cannot be used to access public Dropbox shares?

**Comment by MarineChap** [comment]
For your first comment, sound like good ideas. You were welcome to do a pr. 

For your second comment, it depends on the setting used to create the token. I was using it on public share of Dropbox.  But I did not recreate a token in few years now.
